README.md

```diff
@@ -1,12 +1,160 @@
+
+3. **Ensure dependent services are running:**
+
+   If this microservice depends on other services, make sure they are up and accessible. See [Dependencies](#dependencies) for more details.
+
+4. **Build and run the application:**
+
+   - Using Maven:
+
+     ```bash
+     mvn spring-boot:run
+     ```
+
+   - Using Gradle:
+
+     ```bash
+     gradle bootRun
+     ```
+
+## Dependencies
+
+This microservice depends on the following services:
+
+- **Service Name**
+
+  - **Purpose:** Brief description of what the dependent service does.
+  - **Repository:** [Link to the repository](https://github.com/Sundsvallskommun/service_name)
+  - **Setup Instructions:** Refer to its documentation for installation and configuration steps.
+
+Ensure that these services are running and properly configured before starting this microservice.
+
+## API Documentation
+
+Access the API documentation via Swagger UI:
+
+- **Swagger UI:** [http://localhost:8080/api-docs](http://localhost:8080/api-docs)
+
+Alternatively, refer to the `openapi.yml` file located in the project's root directory for the OpenAPI specification.
+
+## Usage
+
+### API Endpoints
+
+Refer to the [API Documentation](#api-documentation) for detailed information on available endpoints.
+
+### Example Request
+
+```bash
+curl -X GET http://localhost:8080/api/resource
+```
+
+## Configuration
+
+Configuration is crucial for the application to run successfully. Ensure all necessary settings are configured in `application.yml`.
+
+### Key Configuration Parameters
+
+- **Server Port:**
+
+  ```yaml
+  server:
+    port: 8080
+  ```
+
+- **Database Settings:**
+
+  ```yaml
+  spring:
+    datasource:
+      url: jdbc:mysql://localhost:3306/your_database
+      username: your_db_username
+      password: your_db_password
+  ```
+
+- **External Service URLs:**
+
+  ```yaml
+  integration:
+    service:
+      url: http://dependency_service_url
+      oauth2:
+        client-id: some-client-id
+        client-secret: some-client-secret
+
+  service:
+    oauth2:
+      token-url: http://dependecy_service_token_url
+  ```
+
+### Database Initialization
+
+The project is set up with [Flyway](https://github.com/flyway/flyway) for database migrations. Flyway is disabled by default so you will have to enable it to automatically populate the database schema upon application startup.
+
+```yaml
+spring:
+  flyway:
+    enabled: true
+```
+
+- **No additional setup is required** for database initialization, as long as the database connection settings are correctly configured.
+
+### Additional Notes
+
+- **Application Profiles:**
+
+  Use Spring profiles (`dev`, `prod`, etc.) to manage different configurations for different environments.
+
+- **Logging Configuration:**
+
+  Adjust logging levels if necessary.
+
+## Contributing
+
+Contributions are welcome! Please see [CONTRIBUTING.md](CONTRIBUTING.md) for guidelines.
```

Comment:
Reference the file in the .github-repo?

Reply:
It will autoresolve when the PR is merged.